README.md

```diff
@@ -6,51 +6,43 @@ We run an [internal server](./internal-server.md) so private datasets can be kep
 
 We have have general experience [tracking neuroimaging data with git-annex](./git-annex.md).
 
-The data curation scripts should be placed inside the BIDS datasets, under the `code/` folder. See [BIDS specification](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#code). For more convenience, you can create a PR with a curation script that you are working on, so that others can give feedback; once the script is validated, you can simply close the PR and delete the branch without merging.
+# ⚠️ Every dataset should have the following files 
 
-Every dataset should have a README inside it. Example:
-~~~
-# <NAME OF DATASET>
-
-<PLEASE MODIFY>This is an MRI dataset acquired in the context of the XX project. It also contains manual segmentation of MS lesions from expert raters.
-
-Dataset shared by: <NAME AND EMAIL>
-<IF THERE WAS EMAIL COMM>Email communication: <DATE OF EMAIL AND SUBJECT>
-<REPOSITORY OF PROJECT/MODEL, etc>Repository: https://github.com/ivadomed/model_seg_ms_mp2rage
+```
+├── README.md
+├── participants.json
+├── participants.tsv
+├── dataset_description.json
+└── code
+    └── <script_you_used_for_dataset_curation>.py
+```
 
-<IF THERE ARE UPDATES IN THE DATASET:>2022-10-30: Added new data from INsIDER_SCT_Segmentations_COR
+For details, see [BIDS specification](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#code).
 
-## dataset structure
+## `README.md`
 
-participants.tsv: Include the following columns:
-- 'pathology': can take values listed in https://docs.google.com/spreadsheets/d/1yjcA8Z0COn4OZxusIDHjStH2DpeXvscsj-aWE2X-_sg/edit#gid=0
-- 'data_id' is the name center_study_subject used to locate the unprocessed data as under duke/mri/.
-- 'institution_id' is used to identify CenterStudy.
-- others? XXX (Jan pls continue)
+The `README.md` is a [markdown](https://markdown-guide.readthedocs.io/en/latest/index.html) file describing the dataset in more detail.
```

Comment:
I was under the impression that READMEs had to be plain text and just named "README". I'm glad to see [they can be markdown too](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#readme). I'm not sure why I thought this; maybe openneuro.org was using an old version of bids-validator?

Reply:
good to know indeed!

Reply:
This is new, from 8 months ago: [bids-specification#1033](https://github.com/bids-standard/bids-specification/pull/1033).